holoviews/tests/operation/test_downsample.py

```diff
@@ -28,6 +30,46 @@ def test_downsample1d_multi(plottype):
             assert value.size == downsample1d.width
 
 
+def test_downsample1d_shared_data():
+    runs = [0]
+
+    class mocksample(downsample1d):
```

Comment:
Probably would have used `unittest.mock.patch` on the Interface but this is fine.